README.md

```diff
@@ -1 +1,118 @@
+
+5. Create a virtual environment for the USAspending API. In this example we've named the environment *usaspending-api*, but you can call it anything:
+
+        mkvirtualenv usaspending-api
+
+    **Note:** If you're running multiple versions of Python on your machine, you can make sure your environment is running the correct Python version by pointing to a specific binary
+
+        mkvirtualenv --python=[path to installed Python 3.x executable] usaspending-api
+
+6. You should see some output that looks similar to the example below. Essentially, this command creates and activates a new virtualenv named `usaspending-api` with its own set of Python libraries.  Anything you pip install from this point forward will be installed into the *usaspending-api* environment rather than your machine's global Python environment. Your command line prompt indicates which (if any) virtualenv is active.
+
+ **Note:** in the command below, replace `/usr/local/bin/python3.4` with the path to your local Python 3.x executable.
+
+        $ mkvirtualenv --python=/usr/local/bin/python3.4 usaspending-api
+        Running virtualenv with interpreter /usr/local/bin/python3.4
+        New python executable in usaspending-api/bin/python3.4
+        Also creating executable in usaspending-api/bin/python
+        Installing setuptools, pip...done.
+
+        (usaspending-api)$
+
+7. This new environment will be active until you run the `deactivate` command. You can re-activate the environment again at any time by typing `workon usaspending-api`.
+
+### USAspending API Backend
+
+Decide where on your machine you want the USAspending API code to live. From the command line, navigate there and clone the USAspending API repository from GitHub to your local environment:
+
+        $ git clone https://github.com/fedspendingtransparency/usaspending-api.git
+
+Navigate to the USAspending API's main folder:
+
+        $ cd usaspending-api
+
+
+Install the dependencies.  
+
+        $ pip install -r requirements.txt
+
+Next, configure your `local_settings.py` file. This file shouldn't be checked in to source control, but you can see an example in `local_settings.example.py`. Update the database settings to the appropriate database name, user, and password. 
+
+To test your database connection, try running the migrations that come with the project to set up your tables.
+
+        $ python manage.py migrate
+
+You should see output that looks like this:
+
+    (usaspending-api):~/usaspending-api$ ./manage.py migrate
+    Operations to perform:
+    Apply all migrations: admin, auth, awards, contenttypes, sessions
+    Running migrations:
+    Applying awards.0001_initial... OK
+
+Once you've done that, you're ready to start the development server
+
+    $ python manage.py runserver
+
+This will run the application at `0.0.0.0:8000` by default. You can change that by passing the host and port number in to the `runserver` command.
```

Comment:
Minor, but `127.0.0.1` is slightly different than `0.0.0.0` (the latter can be accessed from outside localhost, which is important for virtual machines). This should be the former.